### PR TITLE
Update App Storage/Management and Mobile Apps Live

### PR DESCRIPTION
**Pull request by yuyi-sl**
### Description
- update App Storage documentation
- update Live Mobile App Testing

### Motivation and Context
- Mobile App Management 2.0 is bringing lots of changes to App Storage and general App Management
- new App share feature

### Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation fix (typos, incorrect content, missing content, etc.)

### CODE REVIEW

**Review by alexplischke — APPROVED** [review]
LGTM overall, but would be good to address _some_ of the vale linter warnings in this PR.

File: docs/mobile-apps/automated-testing/appium/appium-2-migration.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ Below you can find an example of a capabilities object for running a test on Sau
 See [Appium 2 for Real devices](/mobile-apps/automated-testing/appium/appium-versions/#real-devices) or [Appium 2 for Virtual devices](/mobile-apps/automated-testing/appium/appium-versions/#virtual-devices) to see which Appium 2 versions are available on Sauce Labs.
 
 :::note
-The below examples are for Mobile Web Testing, for Mobile App Testing please remove the `browserName` and add the [`appium:app`-capability](/mobile-apps/app-storage#using-application-storage-with-automated-test-builds)
+The below examples are for Mobile Web Testing, for Mobile App Testing please remove the `browserName` and add the [`appium:app`-capability](/mobile-apps/app-storage#using-app-storage-with-automated-test-builds)
```
Comment (diemol) [inline docs/mobile-apps/automated-testing/appium/appium-2-migration.md:87]:
```suggestion
The below examples are for Mobile Web Testing, for Mobile App Testing remove the `browserName` and add the [`appium:app`-capability](/mobile-apps/app-storage#using-app-storage-with-automated-test-builds)
```

File: docs/mobile-apps/features/network-capture.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ For Android applications, a minor modification in the manifest is necessary to m
 
 ### Automated Testing
 
-The [App Settings](/mobile-apps/live-testing/live-mobile-app-testing/#app-settings) act as default configurations, but can be overridden by the capabilities specified in your test script. 
+The [App Settings](/mobile-apps/live-testing/live-mobile-app-testing/#app-and-device-settings) act as default configurations, but can be overridden by the capabilities specified in your test script.
```
Comment (diemol) [inline docs/mobile-apps/features/network-capture.md:77]:
```suggestion
The [App Settings](/mobile-apps/live-testing/live-mobile-app-testing/#app-and-device-settings) act as default configurations, but can be overridden by the capabilities specified in your test script.
```

File: docs/mobile-apps/features/network-capture.md [hunk 3]
```diff
@@ -85,7 +85,7 @@ Enable the following capability when you execute your tests.
 
 #### Native Tests (Espresso/XCUITest)
 
-Upload your mobile app to Sauce Labs [through our UI](/mobile-apps/app-storage/#uploading-apps-via-ui) or [our REST API](/mobile-apps/app-storage/#uploading-apps-via-rest-api). 
+Upload your mobile app to Sauce Labs [through our UI](/mobile-apps/app-storage/#upload-apps-via-ui) or [our REST API](/mobile-apps/app-storage/#upload-apps-via-rest-api).
```
Comment (diemol) [inline docs/mobile-apps/features/network-capture.md:88]:
```suggestion
Upload your mobile app to Sauce Labs [through our UI](/mobile-apps/app-storage/#upload-apps-via-ui) or [our REST API](/mobile-apps/app-storage/#upload-apps-via-rest-api).
```

File: docs/mobile-apps/live-testing/live-mobile-app-testing.md [hunk 4]
```diff
@@ -13,101 +13,88 @@ With Sauce Labs, you can test your mobile apps on a variety of Android and iOS/i
...
-| Group Folder Redirect <br/><p><span className="sauceGreen">iOS Only</span></p>             | Enable/disable a group directory redirect. Enabling allows you to use your app's private app container directory instead of the shared app group container directory. When your app gets resigned, the shared directory is not accessible.                                                                                                                                                                                                                                                                                                                                                  |
+Navigate to the **Settings** tab to view the app and device settings.
```
Comment (diemol) [inline docs/mobile-apps/live-testing/live-mobile-app-testing.md:35]:
```suggestion
Navigate to the **Settings** tab to view the app and device settings.
```

File: docs/mobile-apps/live-testing/live-mobile-app-testing.md [hunk 5]
```diff
@@ -13,101 +13,88 @@ With Sauce Labs, you can test your mobile apps on a variety of Android and iOS/i
...
+| Device Language                                                                            | Use the dropdown to select the device language. The language selector will tell your application that the locale of the device and region is set to the selected parameter. You won't need to change the language of the OS manually during a session inside iOS/Android settings. For more information about the locale setting, see the documentation for [iOS](https://developer.apple.com/documentation/foundation/locale) and [Android](https://developer.android.com/reference/java/util/Locale).                        |
```
Comment (diemol) [inline docs/mobile-apps/live-testing/live-mobile-app-testing.md:60]:
```suggestion
| Device Language                                                                            | Use the dropdown list to select the device language. The language selector will tell your application that the device's locale and region are set to the selected parameter. You won't need to manually change the OS's language during a session inside iOS/Android settings. For more information about the locale setting, see the documentation for [iOS](https://developer.apple.com/documentation/foundation/locale) and [Android](https://developer.android.com/reference/java/util/Locale).                        |
```

File: docs/mobile-apps/live-testing/live-mobile-app-testing.md [hunk 6]
```diff
@@ -13,101 +13,88 @@ With Sauce Labs, you can test your mobile apps on a variety of Android and iOS/i
...
+| Device Orientation                                                                         | Use the dropdown to set the device orientation (Landscape or Portrait).                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
```
Comment (diemol) [inline docs/mobile-apps/live-testing/live-mobile-app-testing.md:61]:
```suggestion
| Device Orientation                                                                         | Use the dropdown list to set the device orientation (Landscape or Portrait).                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
```

File: docs/mobile-apps/live-testing/live-mobile-app-testing.md [hunk 7]
```diff
@@ -13,101 +13,88 @@ With Sauce Labs, you can test your mobile apps on a variety of Android and iOS/i
...
+| Device Passcode                                                                            | Enable/disable the device passcode for your apps. If your app requires a device passcode/screenlock to launch, you can enable this setting to run your tests on a passcode-protected device. On Android we are setting 000000, on iOS 089675 as passcode. This is available during Live Testing sessions, see it below in the [Live Testing interface section](/mobile-apps/live-testing/live-mobile-app-testing/#live-test-interface).                                                                                        |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```
Comment (diemol) [inline docs/mobile-apps/live-testing/live-mobile-app-testing.md:63]:
```suggestion
| Device Passcode                                                                            | Enable/disable the device passcode for your apps. If your app requires a device passcode/screenlock to launch, you can enable this setting to run your tests on a passcode-protected device. On Android we are setting 000000, on iOS 089675 as passcode. This feature is available during Live Testing sessions, see it below in the [Live Testing interface section](/mobile-apps/live-testing/live-mobile-app-testing/#live-test-interface).                                                                                        |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```

File: docs/mobile-apps/live-testing/live-mobile-app-testing.md [hunk 8]
```diff
@@ -13,101 +13,88 @@ With Sauce Labs, you can test your mobile apps on a variety of Android and iOS/i
...
+<img src={useBaseUrl('img/live-testing/app-management-ios-example-settings.png')} alt="App settings - iOS" width="800"/>
 
 #### Example Settings - Android
 
-<img src={useBaseUrl('img/live-testing/live-mobile-app-management-android.png')} alt="App settings - Android" width="800"/>
-
+<img src={useBaseUrl('img/live-testing/app-management-android-example-settings.png')} alt="App settings - Android" width="800"/>
 
+## Selecting an App
+### from App Management
```
Comment (diemol) [inline docs/mobile-apps/live-testing/live-mobile-app-testing.md:74]:
```suggestion
### From App Management
```

File: docs/mobile-apps/live-testing/live-mobile-app-testing.md [hunk 9]
```diff
@@ -13,101 +13,88 @@ With Sauce Labs, you can test your mobile apps on a variety of Android and iOS/i
...
+On the main **App Management** page, you can select an app to test by clicking **Start Test** in the row of the desired app group. The app build chosen will be the most recently uploaded build within that group. The device selection page will open, with the option to test on a real device or a virtual device.
```
Comment (diemol) [inline docs/mobile-apps/live-testing/live-mobile-app-testing.md:75]:
```suggestion
On the main **App Management** page, you can select an app to test by clicking **Start Test** in the row of the desired app group. The app build chosen will be the most recently uploaded build in that group. The device selection page will open, with the option to test on a real device or a virtual device.
```

File: docs/mobile-apps/live-testing/testing-apple-pay.md [hunk 10]
```diff
@@ -74,7 +74,7 @@ To make Apple Pay work on Sauce Labs real private devices:
 ## Passcode
 
 One of the Apple Pay requirements is having a set passcode on your phone. Without it, you won't be able to add cards to your wallet. You need to use our Device Passcode capability.
-To initiate a session with automatic Passcode enabling, [explore our passcode capability](/mobile-apps/live-testing/live-mobile-app-testing/#default-app-settings), where you have the option to utilize either a dummy app or our [Sauce Demo application](https://github.com/saucelabs/my-demo-app-ios/releases/tag/2.0.2).
+To initiate a session with automatic Passcode enabling, [explore our passcode capability](/mobile-apps/live-testing/live-mobile-app-testing/#real-device-settings), where you have the option to utilize either a dummy app or our [Sauce Demo application](https://github.com/saucelabs/my-demo-app-ios/releases/tag/2.0.2).
```
Comment (diemol) [inline docs/mobile-apps/live-testing/testing-apple-pay.md:77]:
```suggestion
To initiate a session with automatic Passcode enabling, [explore our passcode capability](/mobile-apps/live-testing/live-mobile-app-testing/#real-device-settings), where you have the option to utilize either a sample app or our [Sauce Demo application](https://github.com/saucelabs/my-demo-app-ios/releases/tag/2.0.2).
```

File: docs/web-apps/automated-testing/electron.md [hunk 11]
```diff
@@ -28,7 +28,7 @@ Sauce Labs currently supports the following test configurations for Electron.
 
 ### App management
 
-Upload a zip file containing your Electron app via [REST API](/mobile-apps/app-storage/#uploading-apps-via-rest-api). You can refer to uploaded apps by either the file ID or file name. Note that as with mobile apps, Electron apps are accessible only to members of the same team, and retained for 60 days.
+Upload a zip file containing your Electron app via [REST API](/mobile-apps/app-storage/#upload-apps-via-rest-api). You can refer to uploaded apps by either the file ID or file name. Note that as with mobile apps, Electron apps are accessible only to members of the same team, and retained for 60 days.
```
Comment (diemol) [inline docs/web-apps/automated-testing/electron.md:31]:
```suggestion
Upload a zip file containing your Electron app via [REST API](/mobile-apps/app-storage/#upload-apps-via-rest-api). You can refer to uploaded apps by either the file ID or filename. Note that as with mobile apps, Electron apps are accessible only to members of the same team, and retained for 60 days.
```

**Review by diemol — APPROVED** [review]
Hey @yuyi-sl, it looks good overall, I just went ahead and made some suggestions shown by the linter.
It'd be good to address all of them, if possible.

**Review by alexplischke — APPROVED** [review]

**Pull request merged**